### PR TITLE
Updating tiddit to v3.6.1

### PR DESCRIPTION
**Pull request by asp8200**
Just updating tiddit to v3.6.1

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - UPD+Chromograph to identify and visualize UPD sites and regions in the chromosomes [#364](https://github.com/nf-core/raredisease/pull/364) and [#366](https://github.com/nf-core/raredisease/pull/366)
 - Added check for presence of case id for each sample in samplesheet [#357](https://github.com/nf-core/raredisease/pull/357)
 
+### Changed
+
+- [#385](https://github.com/nf-core/sarek/pull/385) - Updating tiddit to v3.6.1
```
Comment (ramprasadn) [inline CHANGELOG.md:21]: Could you make this entry under version 1.2.0?

Comment (asp8200) [reply CHANGELOG.md:21]: I'll fix that. When developing on Sarek, we always do a "back to dev" PR as the first thing after each release. The same approach might also be suitable for raredisease.

Comment (asp8200) [reply CHANGELOG.md]: Ah, I see that you do have something like a "back-to-dev" PR.

https://github.com/nf-core/raredisease/pull/384 

It was just a bit late in getting merged. Anyways, no hopefully all good.

Comment (ramprasadn) [reply CHANGELOG.md]: Yup.. 😄

**Review by ramprasadn — APPROVED** [review]
LGTM! 🚀

**Pull request merged**